sql/sql_class.h

```diff
@@ -4458,6 +4463,8 @@ class THD :public Statement,
     table updates from being replicated to other nodes via galera replication.
   */
   bool                      wsrep_ignore_table;
+  /* thread who has started kill for this THD protected by LOCK_thd_data*/
+  my_thread_id              wsrep_aborter;
```

Comment:
Why this needs to be my_thread_id and not just a flag that someone is occupying a critical section ?